content/rules/59br37/proposed.md

```diff
@@ -293,7 +293,7 @@ This [text node][] is [horizontally clipped][] using `text-overflow: ellipsis` a
 
 <a class="example-link" title="Passed Example 3" target="_blank" href="https://w3.org/WAI/content-assets/wcag-act-rules/testcases/59br37/f6c5d3bf4a33699dfa4d53d1949c262df1021735.html">Open in a new tab</a>
 
-This [text node][] is restricted to a single line, by setting a `line-height` that is the same as the `height`. A link to a full version of the poem is also provided.
+This [text node][] is restricted to a single line, by setting a `line-height` that is the same as the `height`. This fulfills expectation 2. A link to a full version of the poem is also provided. The link to the full version is not required by this rule, but is necessary to satisfy [success criterion 1.4.4 Resize text][sc144].
```

Comment:
`[success criterion 1.4.4 Resize text][sc144]` is not converted into a link, see [preview](https://deploy-preview-339--wai-wcag-act-rules.netlify.app/standards-guidelines/act/rules/59br37/proposed/#passed-example-3).

Reply:
Fixed by https://github.com/w3c/wcag-act-rules/pull/339/commits/84d9c6b63919468efe6dbbec0cd4c32e7133683e, thanks @WilcoFiers